cores/arduino/stm32/twi.c

```diff
@@ -101,7 +101,13 @@
   */
 
 /*  Family specific description for I2C */
-#define I2C_NUM (5)
+#if defined(STM32F7xx) || defined(STM32L4xx)
+#define I2C_NUM (4)
+#elif defined(STM32F0xx) || defined(STM32F1xx) || defined(STM32L1xx)
+#define I2C_NUM (2)
+#else // STM32F2xx || STM32F3xx || STM32F4xx || STM32L0xx
+#define I2C_NUM (3)
```

Comment:
What do you thin about 
#elif defined( STM32F2xx) || defined(STM32F3xx) || defined(STM32F4xx) || #defined(STM32L0xx)
#else
#error Unknown Family - unknown I2C_NUM
#endif
so that 3 is not the default for any new family and we will be remonded to complete this part

Reply:
Done!